Crawler/CrawlerService.cs

```diff
@@ -1,95 +1,95 @@
-            Console.WriteLine($"amount of question words: {Normalize(questionWords, wordsCount)}");
+            foreach (var analyzer in analyzers)
+            {
+                DisplayResult(analyzer.Analyze(tokenizedArticle));
+            }
 
-            Console.WriteLine($"average words count between punctuation: {punctuationAnalyzer.CalculateAverageWordsCountBetweenPunctuation(allTokens)}");
-            Console.WriteLine($"max words count between punctuation: {punctuationAnalyzer.CalculateMaxWordsCountBetweenPunctuation(allTokens)}");
-            Console.WriteLine($"standard deviation of words count between punctuation: {punctuationAnalyzer.CalculateWordsCountBetweenPunctuationStandardDeviation(allTokens)}");
-            Console.WriteLine($"ninth decile words count between punctuation: {punctuationAnalyzer.CalculateWordsCountDecile(9, allTokens)}");
-            Console.WriteLine($"second decile words count between punctuation: {punctuationAnalyzer.CalculateWordsCountDecile(2, allTokens)}");
+            applicationLifetime.StopApplication();
+        }
 
-            Console.WriteLine($"count question marks: {Normalize(punctuationAnalyzer.CountCharacter('?', allTokens), wordsCount)}");
-            Console.WriteLine($"count exclamation marks: {Normalize(punctuationAnalyzer.CountCharacter('!', allTokens), wordsCount)}");
-            Console.WriteLine($"count colons: {Normalize(punctuationAnalyzer.CountCharacter(':', allTokens), wordsCount)}");
-            Console.WriteLine($"count dashes: {Normalize(punctuationAnalyzer.CountCharacter('-', allTokens), wordsCount)}");
-            Console.WriteLine($"count quotations mark: {Normalize(punctuationAnalyzer.CountCharacter('"', allTokens)/2 , wordsCount)}");
+        private void DisplayResult(AnalysisResult analysisResult)
+        {
+            var analysisResultAttr = analysisResult.GetType().GetCustomAttribute<AnalysisResultAttribute>();
+            if (analysisResultAttr != null)
+            {
+                Console.WriteLine($"===================={analysisResultAttr.ArticlePart}====================");
+            }
 
-			applicationLifetime.StopApplication();
+            foreach (var prop in analysisResult.GetType().GetProperties())
+            {
+                var resultAttr = prop.GetCustomAttribute<ResultAttribute>();
+                if (resultAttr != null)
+                {
+                    var value = double.Parse(prop.GetValue(analysisResult).ToString());
+
+                    if (prop.GetCustomAttribute<NormalizeAttribute>() != null)
+                    {
+                        value = Normalize(value, analysisResult.AmountOfWords);
+                        Console.Write("(Normalized) ");
+                    }
+
+                    Console.WriteLine($"{resultAttr.Description}: {value:0.##} {resultAttr.Unit}");
+                }
+            }
         }
 
-        public double Normalize(float value, int currentScale)
```

Comment:
I would consider to replace Normalize attribute with an Interface and move this function as default implementation.

Reply:
What will implement it and how it will be used?

Reply:
I thought about it as a class instead of a property, I can't think of a proper way to implement it for props, so for my concerns it's fine.